composer.json

```diff
@@ -24,7 +24,8 @@
   "autoload": {
     "psr-4": {
       "Terminus\\": ["php/", "php/Terminus/"],
-      "Pantheon\\Terminus\\": "src/"
+      "Pantheon\\Terminus\\": "src/",
+      "Pantheon\\Terminus\\Tests\\": "tests/"
```

Comment:
Sorry, misread your code the first time. Corrected my comment, you were right.  Just move the directive to autoload-dev and it's perfect.